### PR TITLE
Feat(Query): Restart Policy On Failure Not Set To 5 for docker compose

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Restart Policy On Failure Not Set To 5 for docker compose.

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "id": "2fc99041-ddad-49d5-853f-e35e70a48391",
+    "queryName": "Restart Policy On Failure Not Set To 5",
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/metadata.json:3]:
```suggestion
    "queryName": "Unrecommended Restart Policy On Failure",
```

Please, also update the query folder name

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/metadata.json (hunk 1 above)
Comment (cxAndreFelicidade) [reply assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/metadata.json:3]: I disagree, I think we should keep the current query name.

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "queryName": "Restart Policy On Failure Not Set To 5",
+    "severity": "MEDIUM",
+    "line": 6,
+    "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json:6]:
```suggestion
    "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  {
+    "queryName": "Restart Policy On Failure Not Set To 5",
+    "severity": "MEDIUM",
+    "line": 17,
+    "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json:12]:
```suggestion
    "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  {
+    "queryName": "Restart Policy On Failure Not Set To 5",
+    "severity": "MEDIUM",
+    "line": 15,
+    "filename": "positive2.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json:18]:
```suggestion
    "fileName": "positive2.yaml"
```

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  {
+    "queryName": "Restart Policy On Failure Not Set To 5",
+    "severity": "MEDIUM",
+    "line": 6,
+    "filename": "positive3.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json:24]:
```suggestion
    "fileName": "positive3.yaml"
```

File: assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  {
+    "queryName": "Restart Policy On Failure Not Set To 5",
+    "severity": "MEDIUM",
+    "line": 17,
+    "filename": "positive4.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/test/positive_expected_result.json:30]:
```suggestion
    "fileName": "positive4.yaml"
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**